packages/datadog/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://github.com/DataDog/helm-charts/releases/download/datadog-2.4.2/datadog-2.4.2.tgz
-packageVersion: 00
+packageVersion: 01
```

Comment:
datadog also seemed to have changes that may have been associated with the charts-build-scripts bump or perhaps some other PR where the packageVersion was not bumped.

e.g. see the following diffs

#### requirements.lock

I also dropped the requirements.lock entirely since charts-build-scripts only uses local chart archives; the requirements.lock is never necessary.

##### main-source

https://github.com/rancher/partner-charts/blob/7738c4a710973985565e4fa8ba171b6276995eb8/charts/datadog/datadog/2.4.200%2Bup2.4.2/requirements.lock#L1-L6

##### main

https://github.com/rancher/partner-charts/blob/7738c4a710973985565e4fa8ba171b6276995eb8/charts/datadog/datadog/2.4.200/requirements.lock#L1-L6

#### requirements.yaml

##### main-source

https://github.com/rancher/partner-charts/blob/7738c4a710973985565e4fa8ba171b6276995eb8/charts/datadog/datadog/2.4.200%2Bup2.4.2/requirements.yaml#L1-L4

##### main

https://github.com/rancher/partner-charts/blob/7738c4a710973985565e4fa8ba171b6276995eb8/charts/datadog/datadog/2.4.200/requirements.yaml#L1-L5